app/services/hacktoberfest_project_importer.rb

```diff
@@ -10,7 +10,7 @@ def import_all(projects)
   def import(project)
     language = import_language(project)
     repository = import_repository(project, language)
-    import_issue(project, repository)
+    import_issue(project, repository) unless repository.banned?
```

Comment:
Am I going insane, but I don't see `Repository#banned?` defined?
In PullRequest, we use `SpamRepositoryService.call(repo_id)`

Reply:
From call, `Repository#banned?` relies on a DB column that we never, ever update anywhere, 'tis always false.

We should just make a call to `SpamRepositoryService#call` as we import these and never use that DB column, we can probably even drop it.

The other part is that whenever this job runs, we should first delete all the existing repos/issues in the DB, so each time this runs we start with a fresh set of imported issues.